--- a/unetbootin.cpp.orig	2015-12-29 23:28:03.268073830 +0000
+++ b/unetbootin.cpp	2015-12-29 23:28:47.229077128 +0000
@@ -3270,7 +3270,7 @@
 	QString configsysText = QString("[menu]\n"
 	"menucolor=15,0\n"
 	"menuitem=windows,Windows\n"
-	"menuitem=grub,"UNETBOOTINB"\n"
+	"menuitem=grub," UNETBOOTINB"\n"
 	"menudefault=windows,30\n"
 	"[grub]\n"
 	"device=ubnldr.exe\n"
@@ -3312,7 +3312,7 @@
 		}
 	}
 	QString bootiniCurText = bootiniCurTextL.join("\n");
-	QString bootiniText = QString("%1\n%2=\""UNETBOOTINB"\"").arg(bootiniCurText).arg(QDir::toNativeSeparators(QString("%1ubnldr.mbr").arg(targetDrive)));
+	QString bootiniText = QString("%1\n%2=\"" UNETBOOTINB"\"").arg(bootiniCurText).arg(QDir::toNativeSeparators(QString("%1ubnldr.mbr").arg(targetDrive)));
 	bootnwOut << bootiniText << endl;
 	if (!QFile::copy(QString("%1bootnw.txt").arg(targetPath), QDir::toNativeSeparators(QString("%1boot.ini").arg(targetDrive))))
 	{
@@ -3330,7 +3330,7 @@
 	QFile vbcdEditF1(QString("%1vbcdedit.bat").arg(targetPath));
 	vbcdEditF1.open(QIODevice::ReadWrite | QIODevice::Text);
 	QTextStream vbcdEditS1(&vbcdEditF1);
-	vbcdEditS1 << QString("bcdedit /create /d \""UNETBOOTINB"\" /application bootsector > %1tmpbcdid").arg(targetPath) << endl;
+	vbcdEditS1 << QString("bcdedit /create /d \"" UNETBOOTINB"\" /application bootsector > %1tmpbcdid").arg(targetPath) << endl;
 	vbcdEditF1.close();
 	if (!warch64)
 		callexternapp(QString("%1vbcdedit.bat").arg(targetPath), "");
@@ -3705,7 +3705,7 @@
 	QString menulstxt = QString(
 	"%9\n\n"
 #ifndef NODEFAULTBOOT
-	"\nmenuentry \""UNETBOOTINB"\" {\n"
+	"\nmenuentry \"" UNETBOOTINB"\" {\n"
 	"\tset root=%8\n"
 	"\t%1 %2 %3 %4\n"
 	"\t%5 %6 %7\n"
@@ -3826,7 +3826,7 @@
 	"timeout 10\n"
 	#endif
 #ifndef NODEFAULTBOOT
-	"\ntitle "UNETBOOTINB"\n"
+	"\ntitle " UNETBOOTINB"\n"
 	#ifdef Q_OS_WIN32
 	"find --set-root %3\n"
 	#endif
@@ -4324,7 +4324,7 @@
 	sdesc4->setText(QString("<b>%1 %2</b>").arg(sdesc4->text()).arg(trcurrent));
 	if (installType == tr("Hard Disk"))
 	{
-		rebootmsgtext->setText(tr("After rebooting, select the "UNETBOOTINB" menu entry to boot.%1\nReboot now?").arg(postinstmsg));
+		rebootmsgtext->setText(tr("After rebooting, select the " UNETBOOTINB" menu entry to boot.%1\nReboot now?").arg(postinstmsg));
 	}
 	if (installType == tr("USB Drive"))
 	{
