diff -uraN unetbootin-source-702/unetbootin.desktop omv-unetbootin-source-702/unetbootin.desktop
--- unetbootin-source-702/unetbootin.desktop	2018-05-09 06:14:10.000000000 +0200
+++ omv-unetbootin-source-702/unetbootin.desktop	2021-02-11 16:18:33.892813335 +0100
@@ -3,7 +3,7 @@
 Name=UNetbootin
 Comment=Tool for creating Live USB drives
 Categories=Application;System;
-Exec=/usr/bin/unetbootin_asroot
+Exec=/usr/bin/xdg-su -c /usr/sbin/unetbootin
 Terminal=false
 Type=Application
 Name[en_US]=UNetbootin
